doc/use-on-icub-head.md

```diff
@@ -0,0 +1,62 @@
+# Use the robotology-superbuild on the icub-head 
+
+## Introduction 
+One of the main goals of the robotology-superbuild is to simplify downloading and building all the software 
+that is necessary to have on the main control computer contained in the head of the iCub robot, whose hostname 
+is `icub-head` (or `pc104` in older iCub versions). This documentation provides the basic information necessary to 
+use the robotology-superbuild on the computer contained in the head of the iCub robot. 
+
+**Warning: if your iCub robot is not currently using the robotology-superbuild, do not attempt to migrate it
+without coordinating with the iCub support.  For any doubt, please open an issue at https://github.com/robotology/icub-support.**
+
+## Prepare your system 
+The operating system contained in the `icub-head`/`pc104` is tipically installed by IIT, for more information on it, please
+check the relevant documentation at http://wiki.icub.org/wiki/The_Linux_on_the_pc104 . On this machine, 
+in `/home/icub/.bashrc_iCub` a script containing several enviroment variables definitions is provided. If you want to  migrate to use
+the robotology-superbuild, you need to remove the existing `.bashrc_iCub` file, and substitute it with the one provided in https://git.robotology.eu/MBrunettini/icub-environment/raw/master/home/bashrc_iCub_superbuild .
+After you modified the `.bashrc_iCub` script, reboot the computer and  in a new shell check che values of the `ROBOTOLOGY_SUPERBUILD_SOURCE_DIR` and `YARP_ROBOT_NAME` env variables.
+If `ROBOTOLOGY_SUPERBUILD_SOURCE_DIR` contains `/usr/local/src/robot/robotology-superbuild`, and `YARP_ROBOT_NAME` contains the string specific to your robot,
+then the modification of the `.bashrc_iCub` was succesful. 
+
+## Get software source code, compile and install
+On the `icub-head`/`pc104`, the software repository necessary to run the iCub robot are contained in the `/usr/local/src/robot` directory.
+On a new machine, this directory should be empty. All the relevant software can download and build with the following commands:
+~~~sh
+cd /usr/local/src/robot
+git clone https://github.com/robotology/robotology-superbuild.git
+cd robotology-superbuild
+mkdir build
+cmake -DROBOTOLOGY_USES_GAZEBO:BOOL=OFF -DROBOTOLOGY_ENABLE_ICUB_HEAD:BOOL=ON ..
```

Comment:
What about `ROBOTOLOGY_ENABLE_DYNAMICS`? isn't `iDynTree` used for estimating the joint torques?

Reply:
The status of iDynTree and the related wholebodydynamics device has always been not really official as part of the iCub ecosystem. Hopefully in one of the next releases we will make it official, and iDynTree will be used for all iCubs, but for now let's keep this instructions simple and similar to the usual setup with which robots are shipped from production.